sources/engine/Stride.Assets/SpriteFont/SystemFontProvider.cs

```diff
@@ -52,72 +51,111 @@ public SystemFontProvider(string fontName)
         /// <inheritdoc/>
         public override FontFace GetFontFace()
         {
-            var factory = new Factory();
+            using var factory = new Factory();
 
-            SharpDX.DirectWrite.Font font;
+            Font font;
             using (var fontCollection = factory.GetSystemFontCollection(false))
             {
-                int index;
-                if (!fontCollection.FindFamilyName(FontName, out index))
+                if (!fontCollection.FindFamilyName(FontName, out var index))
                 {
                     // Lets try to import System.Drawing for old system bitmap fonts (like MS Sans Serif)
                     throw new FontNotFoundException(FontName);
                 }
 
-                using (var fontFamily = fontCollection.GetFontFamily(index))
-                {
-                    var weight = Style.IsBold() ? FontWeight.Bold : FontWeight.Regular;
-                    var style = Style.IsItalic() ? SharpDX.DirectWrite.FontStyle.Italic : SharpDX.DirectWrite.FontStyle.Normal;
-                    font = fontFamily.GetFirstMatchingFont(weight, FontStretch.Normal, style);
-                }
+                using var fontFamily = fontCollection.GetFontFamily(index);
+                var weight = Style.IsBold() ? FontWeight.Bold : FontWeight.Regular;
+                var style = Style.IsItalic() ? SharpDX.DirectWrite.FontStyle.Italic : SharpDX.DirectWrite.FontStyle.Normal;
+                font = fontFamily.GetFirstMatchingFont(weight, FontStretch.Normal, style);
             }
 
             return new FontFace(font);
         }
 
         public override string GetFontPath(AssetCompilerResult result = null)
         {
-            using (var factory = new Factory())
+            if(OperatingSystem.IsWindows())
```

Comment:
Nit: code style, space after opening bracket

Reply:
done?

Reply:
I meant `if (` instead of `if(`

Reply:
Right 🤦 Done, I also updated other if statements